### PR TITLE
[CaptionsIncs] - minor updates

### PR DESCRIPTION
**Pull request by ivandrofly**

### CODE REVIEW

File: libse/SubtitleFormats/CaptionsInc.cs [hunk 1]
```diff
@@ -36,7 +27,7 @@ public static void Save(string fileName, Subtitle subtitle)
                 WriteTime(fs, subtitle.Paragraphs[0].StartTime, false); // first start time
                 WriteTime(fs, subtitle.Paragraphs[subtitle.Paragraphs.Count - 1].EndTime, false); // last end time
 
-                buffer = Encoding.ASCII.GetBytes("Generic Unknown Unknown \"\" Unknown Unknown Unknown                                                                                                                                                                                    ");
```
Comment (ivandrofly) [inline libse/SubtitleFormats/CaptionsInc.cs]: this whole spacing can be achieved with string.padleft

Comment (niksedk) [inline libse/SubtitleFormats/CaptionsInc.cs]: Perhaps "230" instead of "180"?

**Comment by ivandrofly** [comment]
> Perhaps "230" instead of "180"?

https://dotnetfiddle.net/2jTY4a

File: libse/SubtitleFormats/CaptionsInc.cs (hunk 1 above)
Comment (ivandrofly) [inline libse/SubtitleFormats/CaptionsInc.cs]: https://dotnetfiddle.net/2jTY4a /cc @niksedk

Comment (niksedk) [inline libse/SubtitleFormats/CaptionsInc.cs]: 230 is the total width!!!!!

**Pull request closed without merging**